### ISSUE TITLE
plugins are not killed correctly on windows

### ISSUE THREAD

**Issue by cj123** [cj123/assetto-server-manager#358]

**Comment by WraithStar** [comment]
Been doing some testing here. I used KMR and sTracker specifically, and ran plugins with a build using the new plugin launch logic (c424d8f62e6b7978ae9a91891889713931299706)

- if you only use KMR, the plugin will be stopped correctly with no issues.

- if you only use sTracker, it gets messy. Apparently, all of this happens because sTracker for Windows actually runs 2 instances of ``stracker.exe``. As you kill the server, the Manager does kill one of the instances, but for some reason fails to kill the other one, which gets stuck and no matter what you do, the Manager won't just kill it anymore. If you restart the server, the Manager will launch a new instance of stracker that will completely fail to run.

There is also another thing I don't understand, you can see here in the logs the Manager tries to kill the PID 13568, but there was never an sTracker instance that got assigned PID # 13568...

```
time="2019-05-30T17:27:18+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:27:18+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:27:18+02:00" level=error msg="Can't kill process: 13568, err: exit status 128"
time="2019-05-30T17:27:18+02:00" level=error msg="Can't kill process: 13568, err: exit status 128"
```

- if you use both stracker and KMR, the manager will fail to stop sTracker, and for some reason that will prevent KMR from being killed as well. Here is the log

```
time="2019-05-30T17:20:27+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:20:27+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:20:27+02:00" level=error msg="Can't kill process: 12444, err: exit status 128"
time="2019-05-30T17:20:27+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:20:27+02:00" level=error msg="Can't kill process: 12444, err: exit status 128"
time="2019-05-30T17:20:27+02:00" level=error msg="Initial attempt at killing windows process (process.Kill) failed" error="TerminateProcess: Accesso negato."
time="2019-05-30T17:20:27+02:00" level=error msg="Can't kill process: 10716, err: exit status 128"
time="2019-05-30T17:20:27+02:00" level=error msg="Can't kill process: 10716, err: exit status 128"
```

**Comment by WraithStar** [comment]
3ced5a2343b1ee43630e0dfb02c6f317d669fcee fixes this

**Comment by cj123** [comment]
fixed by #400
